crates/completion/src/completions/unqualified_path.rs

```diff
@@ -144,6 +150,7 @@ fn fuzzy_completion(acc: &mut Completions, ctx: &CompletionContext) -> Option<()
         })
     })
     .filter(|(mod_path, _)| mod_path.len() > 1)
+    .take(10)
```

Comment:
That will, most probably, will cause https://github.com/rust-analyzer/rust-analyzer/issues/6594 to reappear: in my experiments, this one had appeared somewhere after 20th result.

Reply:
I've installed your version and fiddling with it, so far we cannot propose `PhantomData` for `phant` input, we need more than 10 here for sure.

Reply:
Hm, it definitelly shows for me:

![image](https://user-images.githubusercontent.com/1711539/102544475-25101000-40b5-11eb-8928-8db90153a69f.png)

Reply:
Aha!

![image](https://user-images.githubusercontent.com/1711539/102544649-67395180-40b5-11eb-874d-699ea4bd9d04.png)

So looks like VS Code rejects some of the completions we are showing. And I've noticed something very interesting as well!

Note how the untaken list contains `std::sync::mpsc::channel`. I think thisshoudn't be a suggestion for phan, because `phan` spanns several path components. cc @jonas-schievink

Reply:
And we need to support `#![doc(hidden)]` (inner) -- thtat's why flt2dec there. rust-analyzer discloses all of the compiler's easer eggs :D